packages/ui/src/SelectField.module.scss

```diff
@@ -95,6 +95,47 @@ $select: ':global .hz-select-field__control';
   }
 }
 
+// when .isMulti is true, we have some tweaks overall
+.multiContainer {
+  #{$select} {
+    padding: c.$grid;
+  }
+
+  :global .hz-select-field__indicators {
+    align-items: flex-start;
+    padding-top: c.$grid;
+  }
+
+  .multiValue {
+    background-color: c.$colorNeutralWhite !important;
+    border: c.$borderWidth solid c.$colorNeutralLight;
+    border-radius: c.$borderRadius;
+    margin: c.$grid * 0.5 !important; // margin collapsing is not there in flexbox. we give half margins to multiValues.
+    height: c.$grid * 5.5; // makes 22px; design is 21px.
```

Comment:
Good thinking! I totally agree with making it a sane increment of `grid` instead of being 100% pixel perfect

Reply:
there're also some 1px mismatches between paddings and what not. (i'll demo this to @pawelklasa for his blessings to the implementation :D designer folks have ultra precise / lazer zoom eyes for these kind of things.)